nixos/modules/services/web-apps/onlyoffice.nix

```diff
@@ -80,6 +80,8 @@ in
       default = "amqp://guest:guest@localhost:5672";
       description = "The Rabbitmq in amqp URI style OnlyOffice should connect to.";
     };
+
+    wopi = lib.mkEnableOption "Enable WOPI support";
```

Comment:
would be nice to describe what WOPI is somewhat, but as https://helpcenter.onlyoffice.com/docs/installation/docs-developer-configuring.aspx#wopi_block doesn't have that either that's fine I suppose